scripts/nmt/bleu.py

```diff
@@ -74,6 +111,12 @@ def compute_bleu(reference_corpus_list, translation_corpus,
             'The number of translations and their references do not match'
 
     for references, translation in zip(zip(*reference_corpus_list), translation_corpus):
+        if bpe:
+            references = list(map(_bpe_to_words, references))
```

Comment:
list comprehension should be preferred in this case. e.g. `[_bpe_to_words(r) for r in references]`

Reply:
I think list(map()) is faster?